acceptance/tests/environment/can_enumerate_environments.rb

```diff
@@ -56,7 +56,7 @@ def curl_master_from(agent, path, headers = '', &block)
 with_puppet_running_on(master, master_opts) do
   agents.each do |agent|
     step "Ensure that an unauthenticated client cannot access the environments list" do
-      on agent, "curl --tlsv1 -ksv https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
+      on agent, "curl --tlsv1 -s -v https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
```

Comment:
@joshcooper I don't follow you. This test also ensures that unauthenticated agents cannot enumerate environments.

Also, since we are on the stable branch, we do not have the ca-cert bundle in place to run this test on AIX regardless of the curl command. Seems like we should skip on Solaris and AIX and address this in the master branch.

Thoughts?